src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothAdapterImpl.cs

```diff
@@ -68,6 +70,26 @@ internal partial class BluetoothAdapterImpl : IDisposable
             }
         }
 
+        internal event EventHandler<AuthenticationRequestedEventArgs> AuthenticationChanged
```

Comment:
This API can not be used in outer. Please call `AuthenticationChanged `in BluetoothAdapter.cs with Browsable option to provide Internal API.

        [EditorBrowsable(EditorBrowsableState.Never)] // This tag makes this API for Internal.
        public static void Enable()
        {
            BluetoothAdapterImpl.Instance.Enable();
        }